dev-ruby/public_suffix/public_suffix-3.1.1-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
This commit can be dropped since all versions of public_suffix now support ruby27.

Reply:
Yes I removed that part from the commit, but there still some changes needed to activate the tests and EAPI 6 -> 7 bump